compat.h

```diff
@@ -0,0 +1,32 @@
+#ifndef UTCP_COMPAT_H
+#define UTCP_COMPAT_H
+
+#ifndef _MSC_VER
+  #include <unistd.h>
+#endif
+
+#ifdef _WIN32
+  #include <winsock2.h>
+#else
+  #include <sys/time.h>
+#endif
+
+#if !defined(PRINT_SIZE_T)
+  #ifdef _WIN32
+    #define PRINT_SIZE_T "%Iu"
+    #define PRINT_SSIZE_T "%Id"
```

Comment:
richtig, in jedem fall muß aber trotzdem verdammt aufgepasst werden, dass sich keine %z wieder einschleichen, da der compiler die unterstützung der windows c library implementierung laut doku nicht prüft, verdammt blöd *hmpf*